### PR TITLE
rework intro, troubleshooting sections

### PR DESCRIPTION
**Pull request by bluegenes**

### CODE REVIEW

**Comment by bluegenes** [comment]
I'll call this PR here - with changes to the intro and troubleshooting sections. Ready for review! 

Tomorrow I'll put in a separate PR with a few minor edits to the rest of the manuscript and attempt to update the conclusions/abstract/author summary sections.

File: content/03.introduction.md [hunk 1]
```diff
@@ -1,27 +1,24 @@
 ## Introduction
-*(draft)*
 
-Sequencing data are now widely available for species across the tree of life, and new sequencing data continues to be generated at an incredible rate [@url:https://www.ncbi.nlm.nih.gov/sra/docs/sragrowth/].
-The wealth of information present in high-throughput sequencing data has already revolutionized our understanding of the diversity and function of organisms and communities, building basic understanding from ecosystems to human health.
+Biological research has become increasingly computational.
```
Comment (taylorreiter) [inline content/03.introduction.md:3]: I love this, as opposed to look there's so much sequencing data

File: content/03.introduction.md [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
+The field of genomics, in particular, has experienced a deluge of high-throughput sequencing data that has already revolutionized our understanding of the diversity and function of organisms and communities, building basic understanding from ecosystems to human health.
+The analysis workflows used to produce these insights often span hundreds of steps and involve a myriad of decisions ranging from small-scale tool and parameter choices to larger-scale decisions around computational experimental design and statistical analyses.
+Each step relies not only upon code written by the user, but on software, its dependencies, and the compute infrastructure and operating system on which the code is executed.
+Historically, this has led to patchwork availability of underlying code for analyses and a lack of interoperability of software and analysis pipelines across compute systems [@doi:10.1038/s41587-020-0439-x].
+Combined with unmet training needs in biological data analysis, these conditions undermine the reuse of data and the reproducibility of biological research, vastly limiting the value of our generated data [@doi:10.1371/journal.pcbi.1005755].
```
Comment (taylorreiter) [inline content/03.introduction.md:8]: yes! love this!

File: content/03.introduction.md [hunk 3]
```diff
@@ -1,27 +1,24 @@
...
-These recommendations have both helped build consensus and fueled additional tool and workflow development.
-Since the latest paper in 2017 [@doi:10.1371/journal.pcbi.1005510], key advancements in workflow scripting, software management, and tools that handle biological data at scale have vastly increased the power of analysis management.
-In particular, the emergence of bioinformatics-focused workflow systems has empowered biologists to analyze biological data at scale.
+The biological research community is strongly committed to addressing these issues, recently formalizing the idea that all life sciences research (including data and analysis workflows) should be Findable, Accessible, Interoperable, and Reusable (FAIR) [@doi:10.1016/j.cels.2018.03.014].
+For computational analyses, these ideals are certainly achievable given current technologies, but implementing them in practice has proven quite difficult, particularly for biologists with insufficient training in computing [@doi:10.1016/j.cels.2018.03.014, @doi:10.1139/facets-2019-0020].
```
Comment (taylorreiter) [inline content/03.introduction.md]:
```suggestion
For computational analyses, these ideals are achievable given current technologies, but implementing them in practice has proven difficult, particularly for biologists with insufficient training in computing [@doi:10.1016/j.cels.2018.03.014, @doi:10.1139/facets-2019-0020].
```

File: content/03.introduction.md [hunk 4]
```diff
@@ -1,27 +1,24 @@
...
-This code can be easily reused and modified to accomodate advances in bioinformatics across diverse workflow system applications, further improving efficiency [@doi:10.1007/s41019-017-0050-4].
+Adopting workflow systems requires some level of up-front investment, first to understand the structure of the system, and then to learn the workflow-specific syntax.
+These challenges can preclude adoption, particularly for researchers without significant computational experience [@doi:10.1139/facets-2019-0020].
+In our experiences with both research and training, these initial learning costs are similar to those required for learning more traditional analysis strategies, but then provide a myriad of additional benefits that both facilitate and accelerate research.
+Furthermore, online communities for sharing reusable workflow code have proliferated, meaning the initial cost of specifying a workflow in a system are mitigated via use and re-use of common steps, leading to faster time-to-insight [@doi:10.1016/j.future.2017.05.041; @doi:10.1007/s41019-017-0050-4].
```
Comment (taylorreiter) [inline content/03.introduction.md]:
```suggestion
Furthermore, online communities for sharing reusable workflow code have proliferated, meaning the initial cost of specifying a workflow in a system is mitigated via use and re-use of common steps, leading to faster time-to-insight [@doi:10.1016/j.future.2017.05.041; @doi:10.1007/s41019-017-0050-4].
```

File: content/07.troubleshooting.md [hunk 5]
```diff
@@ -1,22 +1,70 @@
...
+For users, this can be an incredible advantage.
+Tools under active development are likely to improve software efficiency, add new features, improve user interaction and documentation, and be responsive to queries.
+However, functionality may change as developers build their software, and despite best efforts, documentation can get out of date [@doi:10.1007/s10606-018-9333-1].
+On top of this, it is difficult to grok code and usage errors while unfamiliar with a tool or its underlying language or syntax, especially for researchers without computational training [@doi:10.1371/journal.pcbi.1005755].
+As a result, workflows will invariably require troubleshooting and iteration, often even while running test datasets associated with analysis software.
+The challenges of learning and interacting with research software -- from coding languages to workflow systems to analysis -- require time and patience, and it is important to approach your work with this in mind [@doi:10.1139/facets-2019-0020].
+One of the largest barriers is learning how, when, and where to ask for help.
```
Comment (bluegenes) [inline content/07.troubleshooting.md]:
```suggestion

One of the largest barriers is learning how, when, and where to ask for help.
```

File: content/07.troubleshooting.md [hunk 6]
```diff
@@ -1,22 +1,70 @@
...
+For users, this can be an incredible advantage.
+Tools under active development are likely to improve software efficiency, add new features, improve user interaction and documentation, and be responsive to queries.
+However, functionality may change as developers build their software, and despite best efforts, documentation can get out of date [@doi:10.1007/s10606-018-9333-1].
+On top of this, it is difficult to grok code and usage errors while unfamiliar with a tool or its underlying language or syntax, especially for researchers without computational training [@doi:10.1371/journal.pcbi.1005755].
+As a result, workflows will invariably require troubleshooting and iteration, often even while running test datasets associated with analysis software.
+The challenges of learning and interacting with research software -- from coding languages to workflow systems to analysis -- require time and patience, and it is important to approach your work with this in mind [@doi:10.1139/facets-2019-0020].
```
Comment (taylorreiter) [inline content/07.troubleshooting.md]:
```suggestion
The challenges of learning and interacting with research software -- from coding languages to workflow systems to analysis -- require time and patience [@doi:10.1139/facets-2019-0020].
```

**Review by taylorreiter — COMMENTED** [review]
putting in a pr for some changes to the trouble shooting section

**Pull request merged**